corehq/apps/app_manager/templatetags/xforms_extras.py

```diff
@@ -31,11 +31,11 @@ def trans(name, langs=None, include_lang=True, use_delim=True, prefix=False):
     for lang in langs:
         if lang in name and name[lang]:
             affix = ("" if langs and lang == langs[0] else tag(lang))
-            return affix + name[lang] if prefix else name[lang] + affix
+            return affix + str(name[lang]) if prefix else str(name[lang]) + affix
```

Comment:
just curious if there might ever be unicode values here we need to worry about, in which case str() would cause an error, but unicode() would work